Reason: sphix.environment.NoUri has been deprected and now removed.
sphinx.errors.NoUri is listed as an alternative
Upstream: yes
Uri: https://github.com/mongodb/mongo-c-driver/commit/69eef997e71cb1478b50604f97bb64427e31d59a

From 69eef997e71cb1478b50604f97bb64427e31d59a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Roberto=20C=2E=20S=C3=A1nchez?= <roberto@connexer.com>
Date: Tue, 7 Jul 2020 16:56:11 -0400
Subject: [PATCH] CDRIVER-3509 fix rpm-package-build task

diff --git a/build/sphinx/taglist.py b/build/sphinx/taglist.py
index 954d0502c6..6b324eb16d 100755
--- a/build/sphinx/taglist.py
+++ b/build/sphinx/taglist.py
@@ -24,7 +24,10 @@
 """
 
 from docutils import nodes, utils
-from sphinx.environment import NoUri
+try:
+    from sphinx.environment import NoUri
+except ImportError:
+    from sphinx.errors import NoUri
 try:
     from sphinx.util.compat import Directive
 except ImportError:
